{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"codeError"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/awsappsync\/appsyncclienttypes\/badrequestreason\/codeerror"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes\/BadRequestReason\/codeError","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"codeError"}],"title":"AppSyncClientTypes.BadRequestReason.codeError","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10AWSAppSync03AppB11ClientTypesO16BadRequestReasonO9codeErroryA2EmF","modules":[{"name":"AWSAppSync"}]},"hierarchy":{"paths":[["doc:\/\/AWSAppSync\/documentation\/AWSAppSync","doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes","doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes\/BadRequestReason"]]},"references":{"doc://AWSAppSync/documentation/AWSAppSync/AppSyncClientTypes/BadRequestReason/codeError":{"role":"symbol","title":"AppSyncClientTypes.BadRequestReason.codeError","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"codeError"}],"abstract":[],"identifier":"doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes\/BadRequestReason\/codeError","kind":"symbol","type":"topic","url":"\/documentation\/awsappsync\/appsyncclienttypes\/badrequestreason\/codeerror"},"doc://AWSAppSync/documentation/AWSAppSync/AppSyncClientTypes/BadRequestReason":{"role":"symbol","title":"AppSyncClientTypes.BadRequestReason","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"BadRequestReason"}],"abstract":[{"type":"text","text":"Provides context for the cause of the bad request. The only supported value is CODE_ERROR."}],"identifier":"doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes\/BadRequestReason","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"BadRequestReason"}],"url":"\/documentation\/awsappsync\/appsyncclienttypes\/badrequestreason"},"doc://AWSAppSync/documentation/AWSAppSync/AppSyncClientTypes":{"role":"symbol","title":"AppSyncClientTypes","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"AppSyncClientTypes"}],"abstract":[],"identifier":"doc:\/\/AWSAppSync\/documentation\/AWSAppSync\/AppSyncClientTypes","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"AppSyncClientTypes"}],"url":"\/documentation\/awsappsync\/appsyncclienttypes"},"doc://AWSAppSync/documentation/AWSAppSync":{"role":"collection","title":"AWSAppSync","abstract":[],"identifier":"doc:\/\/AWSAppSync\/documentation\/AWSAppSync","kind":"symbol","type":"topic","url":"\/documentation\/awsappsync"}}}